packages/@okta/i18n/src/properties/login.properties

```diff
@@ -448,6 +448,16 @@ verify.webauthn.biometric.instructions=Your browser or device will prompt you to
 webauthn.biometric.error.factorNotSupported = Security key or biometric authenticator is not supported on this browser. Select another factor or contact your admin for assistance.
 webauthn.biometric.error.factorNotSupported.oneFactor = Security key or biometric authenticator is not supported on this browser. Contact your admin for assistance.
 enroll.webauthn.instructions.noSupportForBiometric = Note: Some browsers may not support biometric authenticators.
+authfactor.webauthn.error.clientdata_parsing_failed=Client data parsing failed
```

Comment:
I guess it's this way mainly because we're following the patten from the v1 messages bundle, where we'll want to be reusing for v1 as well https://github.com/okta/okta-core/pull/51526/files#diff-0c5a124f8509ab561a3675183396cc89b5c5688b20c817c95a6728ef50e6b9a9R360

Reply:
@TianGan-okta Is this the error visible to end user? If yes, then it doesn't make sense, we should avoid blindly copying API errors and display to enduser..

Recently many customers are complaining about such errors.

Reply:
FYI, I asked @vincentvoong-okta and @mattloughran-okta's review on this thread: https://okta.slack.com/archives/C8WSD3TM4/p1617835519328000

Reply:
Even though, why need `authfactor` prefix. `webauthn` shall be good enough?

Reply:
i think @mpeng-okta is following the existing pattern in the okta-core which we are using in V1. If we do have strong opinion here, I can map it to a new key for UI only. The reason why I am using prefix instead of a need key is if will easy for the develop know it is mapped from API, and easy to debug, for example if we get a error key from backend, we just use it's prefix to find the mapped key in login bundle instead of look in to the code and figure out which key is the mapped one.

Reply:
Yeah the current pattern has the full `authfactor.webauthn.error` prefix, and we're sharing keys across OIE and v1, so it would make sense to keep the same keys (not repeating ourselves, not re-translating keys, etc).